plugins/lsp/language-servers/ltex-settings.nix

```
@@ -2,8 +2,18 @@
 with lib;
 {
   enabled =
-    helpers.defaultNullOpts.mkNullable (with types; either bool (listOf str))
-      ''["bibtex" "context" "context.tex" "html" "latex" "markdown" "org" "restructuredtext" "rsweave"]''
+    helpers.defaultNullOpts.mkNullable (with helpers.nixvimTypes; either bool (listOf (maybeRaw str)))
```

```suggestion
    helpers.defaultNullOpts.mkNullableWithRaw (with helpers.nixvimTypes; either bool (listOf (maybeRaw str)))
```

The value itself could be raw too, not just the list elements.

E.g. to use a boolean from some vim global or local variable.

---

done